project_cost_report_supply/static/src/js/report_action.js

```diff
@@ -0,0 +1,79 @@
+/*
+    © 2019 Numigi (tm) and all its contributors (https://bit.ly/numigiens)
+    License LGPL-3.0 or later (http://www.gnu.org/licenses/LGPL.html).
+*/
+odoo.define("project_cost_report_supply.ReportAction", function (require) {
+"use strict";
+
+var core = require("web.core");
+var _t = core._t;
+
+var project_cost_report = require("project_cost_report");
+var ReportAction = project_cost_report.ReportAction;
+var getRecordIdFromEvent = project_cost_report.getRecordIdFromEvent;
+var getRecordNameFromEvent = project_cost_report.getRecordNameFromEvent;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'project_cost_report' is not in camel case. (camelcase)](https://app.codacy.com/app/numigi/odoo-project-addons/pullRequest?prid=4044060)